src/pages/learn/html5/tutorials/crash-course.mdx

```diff
@@ -46,8 +46,6 @@ You'll need a `Dockerfile` to build and run your game server.
 
 <UnfamiliarWithDockerfiles />
 
----
-
 ## Step 2: Integrate Rivet Matchmaker
 
 ### Install the Rivet library
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 55]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Remove version<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/site/129?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->